common/src/service-connectors/matchmaking/matchmaking.types.ts

```diff
@@ -75,6 +76,10 @@ export const MatchmakingSchemas = {
         input: Schemas.GetScrimBySubmissionId_Request,
         output: Schemas.GetScrimBySubmissionId_Response,
     },
+    [MatchmakingEndpoint.ResetScrim]: {
+        input: Schemas.ResetScrim_Request,
+        output: Schemas.ResetScrim_Response,
+    },
```

Comment:
Can we use `MatchmakingEndpoint.ForceUpdateScrimStatus` instead?

Reply:
At a cursory glance it looks like we can, would be better to reuse that

Reply:
https://github.com/SprocketBot/sprocket/pull/106/commits/462640665da7583e04655e7d7cf7ed66b4f4051d